src/components/textarea/isCSSFieldSizingSupported.tsx

```diff
@@ -0,0 +1 @@
+export default () => CSS.supports('field-sizing', 'content')
```

Comment:
Would this be nice as a generic "cssSupported" helper? Just a question, no strong feelings!

Reply:
Personally, I don't think it's worth it. I'm generally in favour of "using the Web" and not abstracting away standard web APIs unless absolutely necessary. In this case, I don't think we'd gain enough to warrant our own abstraction.

Reply:
Like I say, just a thought :-)